test/controllers/tasks/active.psc.details.controller.unit.ts

```diff
@@ -8,13 +11,27 @@ import { getPscs } from "../../../src/services/psc.service";
 import { mockPscList } from "../../mocks/active.psc.details.controller.mock";
 import { urlUtils } from "../../../src/utils/url";
 import { Templates } from "../../../src/types/template.paths";
+import { SectionStatus } from "@companieshouse/api-sdk-node/dist/services/confirmation-statement";
+import { sendUpdate } from "../../../src/utils/update.confirmation.statement.submission";
+import {
+  PEOPLE_WITH_SIGNIFICANT_CONTROL_ERROR,
+  RADIO_BUTTON_VALUE,
+  SECTIONS
+} from "../../../src/utils/constants";
+import { companyAuthenticationMiddleware } from "../../../src/middleware/company.authentication.middleware";
```

Comment:
don't need lines 21,31,32 - the companyAuthenticationMiddleware is mocked and imported by import mocks from "../../mocks/all.middleware.mock";